cicecore/drivers/nuopc/dmi/CICE_RunMod.F90

```diff
@@ -39,12 +39,12 @@ module CICE_RunMod
 !
 !  author Elizabeth C. Hunke, LANL
 !         Philip W. Jones, LANL
-!         William H. Lipscomb, LANL
+         William H. Lipscomb, LANL
```

Comment:
This looks like an error.  You probably need ! in front.

Reply:
Yes your are right.